charts/kube-prometheus-stack/templates/prometheus-operator/deployment.yaml

```
@@ -143,6 +143,11 @@ spec:
             - --web.key-file=/cert/{{ if .Values.prometheusOperator.admissionWebhooks.certManager.enabled }}tls.key{{ else }}key{{ end }}
             - --web.listen-address=:{{ .Values.prometheusOperator.tls.internalPort }}
             - --web.tls-min-version={{ .Values.prometheusOperator.tls.tlsMinVersion }}
+            {{- if .Values.prometheusOperator.extraArgs }}
+            {{- range .Values.prometheusOperator.extraArgs }}
+            - {{ . }}
+            {{- end }}
+            {{- end }}
```

```suggestion
            {{- with.Values.prometheusOperator.extraArgs }}
            {{- tpl (toYaml .) $ | nindent 12 }}
            {{- end }}
```

That should be worked as well, or I'm wrong?

---

Agreed. Will add above